RecoTracker/MkFitCore/src/HitStructures.cc

```diff
@@ -1,7 +1,7 @@
 #include "RecoTracker/MkFitCore/interface/HitStructures.h"
 
 #include "RecoTracker/MkFitCore/interface/IterationConfig.h"
-
+#include "RecoTracker/MkFitCore/src/Matriplex/Memory.h"
```

Comment:
we usually include the local directory private files (not from /interface) directly
```suggestion
#include "Matriplex/Memory.h"
```